src/main/java/org/jabref/gui/remote/JabRefMessageHandler.java

```diff
@@ -18,20 +17,17 @@
 
     @Override
     public void handleCommandLineArguments(String[] message) {
-        try {
-            ArgumentProcessor argumentProcessor = new ArgumentProcessor(message, ArgumentProcessor.Mode.REMOTE_START);
-
-            List<ParserResult> loaded = argumentProcessor.getParserResults();
-            for (int i = 0; i < loaded.size(); i++) {
-                ParserResult pr = loaded.get(i);
-                boolean focusPanel = i == 0;
-                Platform.runLater(() ->
-                        // Need to run this on the JavaFX thread
-                        JabRefGUI.getMainFrame().addParserResult(pr, focusPanel)
-                );
-            }
-        } catch (ParseException e) {
-            LOGGER.error("Error when parsing CLI args", e);
+
+        ArgumentProcessor argumentProcessor = new ArgumentProcessor(message, ArgumentProcessor.Mode.REMOTE_START);
```

Comment:
Now we have the problem: /home/travis/build/JabRef/jabref/src/main/java/org/jabref/gui/remote/JabRefMessageHandler.java:21: error: unreported exception ParseException; must be caught or declared to be thrown
        ArgumentProcessor argumentProcessor = new ArgumentProcessor(message, ArgumentProcessor.Mode.REMOTE_START);

Reply:
This is totally odd. I first replaced it with our ParseException. Then it complained that the exception is never thrown. So I removed it. Makes no sense

Reply:
It does not throw our Parse Exception but the one from apache cli.